docs/dev/libs/state_processor_api.md

```diff
@@ -256,9 +258,11 @@ public class StatefulFunctionWithTime extends KeyedProcessFunction<Integer, Inte
 public class StatefulFunctionWithTime extends KeyedProcessFunction[Integer, Integer, Void] {
  
   var state: ValueState[Integer];
+  var stateDescriptor: ValueStateDescriptor[Integer];
  
    @throws[Exception]
```

Comment:
Just curious, why we add `@throws[Exception]` here, where will the Exception throw from?